### ISSUE TITLE
Make dev-tool a global command

### ISSUE THREAD

**Issue by roxblnfk** [yiisoft/yii-dev#210]
It will be legendary if yii-dev-tool could work as global command.

### For example

We have a Foo project that contains a lot of repositories.

Right now we have some cloned repositories:

```
- git
  - foo
    - core
    - contracts
    - bar
```

We are running

```bash
cd git/foo
yii-dev init
```

The dev tool will ask some questions about project and make some config file(s) in the `git/foo` folder.
And now we can use here the tool to clone more project repositories, update the existing and use ither features.

What do you think about the proposal?

**Comment by viktorprogger** [comment]
I'd prefer to install dev-tool as a dev dependency and run `vendor/bin/dev-tool init`. This approach will force it to work in any project or set of packages.

**Comment by roxblnfk** [comment]
@viktorprogger i think it possible to use dev-tool in a project that placed in a subfolder of the inited "super project".

**Comment by viktorprogger** [comment]
Yes, that's bad. It doesn't work with docker volumes. When you try to work with one of the sub-projects (e.g. start a server for the yiisoft/demo), it's fucking annoying to find out all the workarounds you have to do for that.
